docs/app/(home)/components/use-cases-server.tsx

```diff
@@ -42,9 +42,9 @@ async function callAPI(endpoint) {
 
   if (response.status === 429) {
     const retryAfter = response.headers.get('Retry-After');
-    // Customize the duration before retrying
+    // Customize the duration before retrying (assume "Retry-After" is an ISO string)
```

Comment:
retry after header js meant to be either seconds, or a HTTP Date format - not ISO

https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/Retry-After